packages/eslint-config-hypothesis/react.js

```diff
@@ -18,6 +18,7 @@ export default [
       'react-hooks/exhaustive-deps': 'error',
 
       // Prop validation should be performed by TypeScript/JSDoc.
+      'react/no-unknown-property': 'off',
```

Comment:
The comment above is relevant (and in fact the reason) to disable this rule too.

I'm adding it before `react/prop-types` to keep them alphabetically ordered.